azblob/zc_credential_shared_key.go

```diff
@@ -39,6 +39,14 @@ func (f SharedKeyCredential) AccountName() string {
 	return f.accountName
 }
 
+func (f SharedKeyCredential) getAccountKey() []byte {
+	return f.accountKey
+}
+
+func (f SharedKeyCredential) getUDKParams() *UserDelegationKey {
```

Comment:
I wonder if it would be helpful to include a comment here to say that this method is present to satisfy the StorageCredential interface. Unless it's understood in Go that basically no-op methods like this are generally added to satisfy an interface.